yjit/src/codegen.rs

```diff
@@ -7711,6 +7716,7 @@ fn gen_send_iseq(
 
     if forwarding_call {
         assert_eq!(1, num_params);
+        asm.ctx.dealloc_temp_reg(asm.ctx.get_stack_size());
```

Comment:
Is this load-bearing? Existing uses are close to where the store actually is, and this seems to have no relation to what follows.
If it is deallocating for `(-1)`, can you extract `asm.stack_opnd(-1)` as a variable and use `asm.ctx.dealloc_temp_reg(ci_opnd.stack_idx())` instead?

TBH I'm not too sure how reg alloc for negative slots work and the sign to unsigned cast is confusing.

Reply:
This is what I suggested during pairing since `asm.spill_temps()` ignores negative indexes. Without this, it would be written only to a register and the interpreter can't see it on the fallback.